Tools/Pipeline/Windows/MainView.cs

```diff
@@ -409,7 +413,8 @@ public void AddTreeItem(IProjectItem item)
             node.ImageIndex = iconIdx;
             node.SelectedImageIndex = iconIdx;
 
-            _treeView.SelectedNode = node;
+            if (!_projectLoading)
+                _treeView.SelectedNode = node;
```

Comment:
Why does this work to ensure the project is selected?  Is it because you are disabling selection of the child items during project loading?